packages/pn-personagiuridica-webapp/src/navigation/routes.tsx

```diff
@@ -130,6 +133,19 @@ function Router() {
                     </PrivateRoute>
                   }
                 />
+                <Route
+                  path={routes.INTEGRAZIONE_API}
+                  element={
+                    <PrivateRoute
+                      currentRoles={[]}
```

Comment:
Same considerations done for the menu voice

Reply:
As decided in SU, this page must be shown to all users